dipy/data/__init__.py

```diff
@@ -313,3 +313,38 @@ def matlab_life_results():
     matlab_rmse = np.load(pjoin(DATA_DIR, 'life_matlab_rmse.npy'))
     matlab_weights = np.load(pjoin(DATA_DIR, 'life_matlab_weights.npy'))
     return matlab_rmse, matlab_weights
+
+
+def hermite_sdp_constraints(radial_order):
+    """Import semidefinite programming constraint matrices to enforce
+    sum-of-squares constraints on Hermite polynomials, used for example in the
+    case of MAP-MRI, and generated as described in [1]_.
+
+    Parameters
+    ----------
+    radial_order : unsigned int,
+        An even integer that represent the order of the basis
+
+    Returns
+    -------
+    sdp_constraints : array
+        Constraint matrices
+
+    References
+    ----------
+    .. [1] Dela Haije et al. "Enforcing necessary non-negativity constraints
+           for common diffusion MRI models using sum of squares programming".
+           NeuroImage 209, 2020, 116405.
+
+    """
+
+    mf = 'hermite_constraint_' + str(radial_order) + '.csv'
+    coo = np.loadtxt(pjoin(DATA_DIR, mf), delimiter=",")
```

Comment:
You should check if the file exists in case someone put a wrong radial order. 
Otherwise, you can add a check ` if 0<= radial_order <= 8`

Reply:
`radial_order` is checked in reconst/mapmri.py when a MapmriModel object is created, which I believe should be the only time these files are loaded. Should there be an additional check in data/__init__.py? If so, what kind of error should be raised when the file is missing? (None of the other defs in data/__init__.py have such checks.)

Reply:
Indeed, good point.

However, I think you should add an additional test `if radial_order > 8`  in `data/__init__.py`.

Some people might do `radial_order=10` which will pass the `MapmriModel` test but fail on this function.

you can raise a `ValueError`